lua/autorun/client/cl_cfc_help.lua

```diff
@@ -10,31 +51,41 @@ local function openHelp()
     end
 
     window:Center()
-    window:SetTitle( "CFC Help" )
+    window:SetTitle( title )
     window:MakePopup()
 
     window.Paint = function( self, w, h )
-        draw.RoundedBox( 8, 0, 0, w, h, uiColor )
+        draw.RoundedBox( 8, 0, 0, w, h, self.colors.ui )
     end
 
     local html = vgui.Create( "DHTML", window )
 
     html:Dock( FILL )
-    html:OpenURL( "https://cfcservers.org/learn/player/" )
+    html:OpenURL( url )
 end
 
 local function hideScrollbar()
     -- WIP
 end
 
-concommand.Add( "cfc_help", openHelp )
+hook.Add( "OnPlayerChat", "CFC_Help_CommandMatcher", function( ply, msg )
+    local commandInfo = CFCHelp.commandMap[msg]
 
-hook.Add( "OnPlayerChat", "CFC_Help_OpenHelpCommand", function( ply, msg )
-    if not string.StartWith( msg, "!help" ) then return end
+    if not commandInfo then return end
 
     if ply == LocalPlayer() then
-        openHelp()
+        local helpType = commandInfo.helpType
+
+        if helpType == "html" then
+            CFCHelp:openUrl( commandInfo.url, commandInfo.title )
+        end
+
+        if helpType == "func" then
```

Comment:
```suggestion
        if helpType == "function" then
```
Your data structure has this set as `function`